### PR TITLE
Update the documentation about support to Open Y 2.0

### PR DESCRIPTION
**Pull request by AndreyMaximov**

### CODE REVIEW

**Review by ddrozdik — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -39,20 +39,40 @@ composer create-project ymcatwincities/openy-project MY_PROJECT --no-interaction
...
+#### Latest DEVELOPMENT version (1.x)
 ```
 composer create-project ymcatwincities/openy-project:8.1.x-development-dev MY_PROJECT --no-interaction --no-dev
 ```
 
 This command will build project based on [**latest development**](https://github.com/ymcatwincities/openy/commits/8.x-1.x) release.
 
+#### Latest DEVELOPMENT version (2.x)
+```
+composer create-project ymcatwincities/openy-project:8.2.x-development-dev MY_PROJECT --no-interaction --no-dev
```
Comment (podarok) [inline README.md:61]: ```Installing drupal/core (8.4.0-alpha1): Loading from cache```
That's why we have https://github.com/ymcatwincities/openy#installation-of-open-y ```composer update``` command in a queue after composer create finished

File: README.md [hunk 2]
```diff
@@ -39,20 +39,40 @@ composer create-project ymcatwincities/openy-project MY_PROJECT --no-interaction
 
 This command will build project based on [**latest stable**](https://github.com/ymcatwincities/openy/releases) release.
 
-#### Latest DEVELOPMENT version
+#### Latest 2.0 BETA version
+```
+composer create-project ymcatwincities/openy-project:8.2.x-dev MY_PROJECT --no-interaction --no-dev
```
Comment (podarok) [inline README.md:45]: ```Installing drupal/core (8.4.0-alpha1): Loading from cache```
That's why we have https://github.com/ymcatwincities/openy#installation-of-open-y ```composer update``` command in a queue after composer create finished

File: README.md [hunk 3]
```diff
@@ -39,20 +39,40 @@ composer create-project ymcatwincities/openy-project MY_PROJECT --no-interaction
...
 
 You should use composer command without `--no-dev` if you would like to get environment that was configured especially for OpenY. So it should look like this:
 ```
 composer create-project ymcatwincities/openy-project:8.1.x-development-dev MY_PROJECT --no-interaction
 ```
 
+or (for Open Y 2.0)
+
+```
+composer create-project ymcatwincities/openy-project:8.2.x-development-dev MY_PROJECT --no-interaction
```
Comment (podarok) [inline README.md:78]: ```Installing drupal/core (8.4.0-alpha1): Loading from cache```
That's why we have https://github.com/ymcatwincities/openy#installation-of-open-y ```composer update``` command in a queue after composer create finished

File: README.md [hunk 4]
```diff
@@ -39,20 +39,40 @@ composer create-project ymcatwincities/openy-project MY_PROJECT --no-interaction
...
+composer create-project ymcatwincities/openy-project:8.2.x-development-dev MY_PROJECT --no-interaction --no-dev
+```
+
+This command will build project based on the [**development branch**](https://github.com/ymcatwincities/openy/commits/8.x-2.x) release.
+
 ## Development environment
 
 You should use composer command without `--no-dev` if you would like to get environment that was configured especially for OpenY. So it should look like this:
 ```
 composer create-project ymcatwincities/openy-project:8.1.x-development-dev MY_PROJECT --no-interaction
```
Comment (podarok) [inline README.md:71]: ```Installing drupal/core (8.4.0-alpha1): Loading from cache```
That's why we have https://github.com/ymcatwincities/openy#installation-of-open-y ```composer update``` command in a queue after composer create finished

File: README.md [hunk 5]
```diff
@@ -39,20 +39,40 @@ composer create-project ymcatwincities/openy-project MY_PROJECT --no-interaction
...
+#### Latest 2.0 BETA version
+```
+composer create-project ymcatwincities/openy-project:8.2.x-dev MY_PROJECT --no-interaction --no-dev
+```
+
+This command will build project based on [**latest 8.2 beta**](https://github.com/ymcatwincities/openy/releases) release.
+
+#### Latest DEVELOPMENT version (1.x)
 ```
 composer create-project ymcatwincities/openy-project:8.1.x-development-dev MY_PROJECT --no-interaction --no-dev
```
Comment (podarok) [inline README.md:53]: ```Installing drupal/core (8.4.0-alpha1): Loading from cache```
That's why we have https://github.com/ymcatwincities/openy#installation-of-open-y ```composer update``` command in a queue after composer create finished

**Review by podarok — CHANGES_REQUESTED** [review]
1 - fix docs or composer, please

**Review by podarok — APPROVED** [review]

**Pull request merged**